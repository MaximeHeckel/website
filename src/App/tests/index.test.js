import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { App } from '../';

it('renders correctly', () => {
  const tree = renderer.create(
    <App />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});