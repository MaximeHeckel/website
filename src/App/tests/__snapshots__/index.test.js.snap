// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="main-layout"
>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      <img
        alt="logo"
        src="logo.png"
      />
    </div>
    <div
      className={undefined}
    >
      <button
        className="undefined undefined"
        onClick={[Function]}
      >
        More
      </button>
      <button
        className="undefined undefined"
        onClick={[Function]}
      >
        Contact Me
      </button>
    </div>
  </div>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      Hi, I'm Maxime
    </div>
    <div
      className={undefined}
    >
      Iâ€™m a San Francisco
        based software engineer and space
        enthusiast, currently working at Docker. I like playing
        with Golang, Node.JS, React, Redux and containers.
    </div>
    <div
      className={undefined}
    >
      <a
        className="undefined undefined"
        href="https://twitter.com/MaximeHeckel"
        rel="noopener noreferrer"
        target="_blank"
      >
        twitter
      </a>
      <a
        className="undefined undefined"
        href="https://medium.com/@MaximeHeckel"
        rel="noopener noreferrer"
        target="_blank"
      >
        medium
      </a>
      <a
        className="undefined undefined"
        href="https://www.instagram.com/maximeheckel/"
        rel="noopener noreferrer"
        target="_blank"
      >
        instagram
      </a>
      <a
        className="undefined undefined"
        href="https://health.maximeheckel.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        health
      </a>
      <a
        className="undefined undefined"
        href="https://github.com/MaximeHeckel"
        rel="noopener noreferrer"
        target="_blank"
      >
        github
      </a>
    </div>
  </div>
</div>
`;
